score-lang-cli/src/main/java/org/openscore/lang/cli/SlangCLI.java

```diff
@@ -72,9 +73,9 @@ public String run(
         if (!triggerAsync) {
             StopWatch stopWatch = new StopWatch();
             stopWatch.start();
-            id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties);
+            id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties, quiet);
             stopWatch.stop();
-            return triggerSyncMsg(id, stopWatch.toString());
+            return (!quiet) ? triggerAsyncMsg(id, compilationArtifact.getExecutionPlan().getName()) : "";
```

Comment:
you changed the method name from triggerSyncMsg to triggerAsyncMsg ? why? it's to diff methods?